wp-content/plugins/buddyforms/assets/js/buddyforms.js

```diff
@@ -263,30 +263,30 @@ function BuddyForms() {
             hint_html += '</small>';
             passwordHint.remove();
             strengthResult.html('');
-            switch (strength.score) {
-                case 0:
-                case 1:
-                    strengthResult.addClass('short').html(buddyformsGlobal.pwsL10n.short);
-                    break;
-                case 2:
-                    strengthResult.addClass('bad').html(buddyformsGlobal.pwsL10n.bad);
-                    break;
-
-                case 3:
-                    strengthResult.addClass('good').html(buddyformsGlobal.pwsL10n.good);
-                    break;
-
-                case 4:
-                    strengthResult.addClass('strong').html(buddyformsGlobal.pwsL10n.strong);
-                    break;
-
-                case 5:
-                    strengthResult.addClass('short').html(buddyformsGlobal.pwsL10n.mismatch);
-                    break;
-
-                default:
-                    strengthResult.addClass('short').html(buddyformsGlobal.pwsL10n.short);
+            if (strength === 5) {
```

Comment:
Ok.

Reply:
Add comment "handle password mismatch" for this statement.